Assets/_BForBoss/_Utility/Scripts/PanicHelper.cs

```diff
@@ -18,5 +19,23 @@ public static void Panic(Exception exception)
             Application.Quit(1);
 #endif
         }
+        
+        public static void PanicIfNullObject<T>(this T effectedClass, System.Object fieldToCheck, string fieldName) where T : class
```

Comment:
I made these extension methods purely so that people don't need to enter the class type as a parameter (i.e. less typing). Can change if people are not a fan.